### DISCUSSION TITLE
PSA: sync with lots of merge commits

### DISCUSSION THREAD

**Discussion by tesseralis** [reactjs/reactjs.org-translation#246] [Team Posts]
PSA that there was recently a pull request merged that turned all the http links to https: https://github.com/reactjs/reactjs.org/pull/1644

This might end up causing a lot of sync errors, especially if a lot of content is already translated. Changing all the http to https should be enough to fix it.

**Comment by asantarissy** [comment]
Thanks @tesseralis, noted 😄
